lib/metabox-api.php

```diff
@@ -0,0 +1,383 @@
+<?php
+/**
+ * Initialization and wp-admin integration for the Gutenberg editor plugin.
+ *
+ * @package gutenberg
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'Silence is golden.' );
+}
+
+function gutenberg_meta_box_api() {
```

Comment:
The "API" naming here could be misleading. It's not really much of an API (at least in the RESTful sense), more just a partial page.

Reply:
gutenberg_metabox_partial_page()?